packages/rpc/src/__tests__/HttpClient.test.ts

```diff
@@ -1001,6 +1050,67 @@ describe(HttpClient, () => {
       expect(await client.sendRequest('cf_account_info', BROKER_ACCOUNT_ID)).toMatchSnapshot();
     });
 
+    it('gets vault swap data for bitcoin', async () => {
+      expect(
+        await client.sendRequest(
+          'broker_request_swap_parameter_encoding',
+          { chain: 'Bitcoin', asset: 'BTC' },
+          { chain: 'Ethereum', asset: 'USDC' },
+          '0x4838B106FCe9647Bdf1E7877BF73cE8B0BAD5f97',
+          10,
+          {
+            chain: 'Bitcoin',
+            min_output_amount: '0x1',
+            retry_duration: 100,
+          },
+        ),
+      ).toMatchSnapshot();
```

Comment:
i think we have inline snapshots here no? i would prefer to use them unless they are massive.

Reply:
sounds good